tests/testapps/CTCatalog/CTCatalog/Tests/CTCLineTestViewController.mm

```diff
@@ -28,8 +28,6 @@ @implementation CTTruncatedLineTestView {
 }
 
 - (void)drawRect:(CGRect)rect {
-    UIColor* color = [UIColor blueColor];
```

Comment:
it may have been another file? I'm not sure

---
In reply to: [78249523](https://github.com/Microsoft/WinObjC/pull/989#discussion_r78249523) [](ancestors = 78249523)